samples/arduino/starterkit/Spaceship2.js

```diff
@@ -18,54 +18,43 @@ var gpio = require('gpio');
 var pins = require('arduino101_pins');
 
 // set up the GPIO pins
-var button = gpio.open({
-    pin: pins.IO2,
-    direction: 'in',
-    edge: 'any'
-});
-var led1 = gpio.open({
-    pin: pins.IO4,
-    direction: 'out'
-});
-var led2 = gpio.open({
-    pin: pins.IO7,
-    direction: 'out'
-});
-var led3 = gpio.open({
-    pin: pins.IO8,
-    direction: 'out'
-});
-
-var btnstate = false;
+var button = gpio.open({pin: pins.IO2, direction: 'in', edge: 'rising'});
```

Comment:
No actually this is intentional but changes the behavior a bit. It will immediately turn on when it gets a rising edge but shuts off at an even 250ms after that. This is a safer way to code because if one edge is missed it will correct itself on the next one. And since it keeps checking until the value is low it will always eventually shut off. The other way would have potentially left the lasers on indefinitely when there was a missed edge.